### PR TITLE
iOS

### PR DESCRIPTION
**Pull request by turtledreams**

### CODE REVIEW

File: ios/next.md [hunk 1]
```diff
@@ -95,8 +95,8 @@
 </p>
 <div class="callout callout--info">
   <p>
-    If you are in doubt about the correctness of your Countly SDK integration
-    you can learn about methods to verify it from
+    If you are in doubt about the correctness of your Countly SDK integration,
+    you can learn about methods and verification from
```
Comment (turtledreams) [inline ios/next.md:99]: this does not sound right. methods are verification methods. nothing more. if you want to eliminate sth eliminate the methods instead?

File: ios/next.md [hunk 2]
```diff
@@ -3749,7 +3749,7 @@ config.eventSendThreshold = 1</code></pre>
   </p>
 </div>
 <p>
-  <span style="font-weight: 400;">You can integrate the Countly iOS SDK using CocoaPods. For more information, please see the </span><a href="https://cocoapods.org/pods/Countly"><span style="font-weight: 400;">Countly CocoaPods page</span></a><span style="font-weight: 400;">. Please ensure you have the latest version of CocoaPods and your local spec repo is updated. For Notification Service Extension targets, please ensure your Podfile uses something similar to the following sub specs:</span>
+  <span style="font-weight: 400;">You can integrate the Countly iOS SDK using CocoaPods. For more information, please see the </span><a href="https://cocoapods.org/pods/Countly"><span style="font-weight: 400;">Countly CocoaPods page</span></a><span style="font-weight: 400;">. Please ensure you have the latest version of CocoaPods and your local spec repo is updated. For Notification service extension targets, please ensure your Podfile uses something similar to the following sub specs:</span>
```
Comment (turtledreams) [inline ios/next.md:3752]: it seems to be a thing https://developer.apple.com/documentation/usernotifications/unnotificationserviceextension

File: ios/next.md (hunk 1 above)
Comment (AbhisekGs) [reply ios/next.md:99]: got it, you can learn about verification - it should be

File: ios/next.md (hunk 2 above)
Comment (AbhisekGs) [reply ios/next.md:3752]: oops will change it right away.

Comment (AbhisekGs) [reply ios/next.md]: done boss!

File: ios/next.md (hunk 1 above)
Comment (AbhisekGs) [reply ios/next.md]: done boss!

**Review by ArtursKadikis — APPROVED** [review]

**Pull request merged**